sdk/include/aditof/status_definitions.h

```diff
@@ -15,6 +15,7 @@ enum class Status {
     BUSY,             //!< Device or resource is busy
     UNREACHABLE,      //!< Device or resource is unreachable
     INVALID_ARGUMENT, //!< Invalid arguments provided
+    NOT_AVAILABLE,    //!< The requested feature is not available
```

Comment:
A slightly shorter alternative is: UNAVAILABLE
The description could be more generic: "The requested action or resource is unavailable"

Reply:
done.